pandoc-types.cabal

```diff
@@ -1,5 +1,5 @@
 Name:                pandoc-types
-Version:             1.17.3
+Version:             1.17.4
```

Comment:
jgm will increment the version number in a separate commit when he releases a new version... no need to do it in a pull request. (same for the changelogs AFAIK)